modules/gatsby/bin/ocular.js

```diff
@@ -121,8 +137,17 @@ const commands = {
           const file = readFileSync(`${TEMPLATE_DIR}/${filename}`);
           console.log('Writing', `${DIR_PATH}/${filename}`, file.slice(20))
           writeFileSync(`${DIR_PATH}/${filename}`, file);
+        }        
+        const UPDATED_PACKAGE_JSON = require(`${DIR_PATH}/package.json`)
```

Comment:
Is it possible to just merge the new properties in without removing others that might already be present?

Reply:
i may do a different PR for that.